<!DOCTYPE html>
<!-- Latest compiled and minified CSS -->
<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css"> -->

<!-- Optional theme -->
<!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css"> -->

<!-- Latest compiled and minified JavaScript -->
<!-- <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script> -->


<html lang="en">
<head>

  <link rel="stylesheet" type="text/css" href="cyborg.css">
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="">
  <meta name="author" content="">
  <link rel="icon" href="../../favicon.ico">

  <title>Jacob Schreiber</title>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-86984318-3"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-86984318-3');
</script>

<link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet'>
<style>
body {
    font-family: 'Raleway';font-size: 22px;
}

/* unvisited link */
a:link {
  color: hotpink;
}

/* visited link */
a:visited {
  color: hotpink;
}

/* mouse over link */
a:hover {
  color: grey;
}

/* selected link */
a:active {
  color: hotpink;
}

b {
  color: hotpink;
}

.projbutton {
    border: none;
    color: white;
    padding: 4px 12px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 14px;
    margin: 4px 2px;
    -webkit-transition-duration: 0.4s; /* Safari */
    transition-duration: 0.4s;
    cursor: pointer;
    border-radius: 8px;
    font-family: 'Raleway';
}

.projbutton:hover {
  background-color: #4CAF5000;
}

.button1{background-color:#B8204D;border:2px solid #B8204D}
.button2{background-color:#50c473;border:2px solid #50c473}
.button3{background-color:#A61A1D;border:2px solid #A61A1D}
.button4{background-color:#3499CD;border:2px solid #3499CD}
.button5{background-color:#40e0d0;border:2px solid #40e0d0}
.button6{background-color:#ffb347;border:2px solid #ffb347}

</style>
</head>

<div class="banner">
</div>
</div>

<body role="document" style='font="raleway"'>
  <div class="container theme-showcase" role="main">
    <!-- Main jumbotron for a primary marketing message or call to action -->
    <div class="jumbotron">
      <p style="font-size:16px">
        <img src="jmschr_corgi.jpg" style="border-radius:50%; height:160px" align="left">

        <span>
        <br>
        <b>
          &emsp;&emsp;
          <a href="Schreiber__Jacob_CV.pdf" target="_blank">CV</a> |
          <a href="#publications">PUBLICATIONS</a> | 
          <a href="#talks">TALKS</a> | 
          <a href="#employment">EMPLOYMENT</a> | 
          <a href="palleros.html" target="_blank">DR. PALLEROS</a> | 

          <a href="http://github.com/jmschrei" target="_blank"><img src="logos/Octocat.png" style="height:30px"></a> 
          <a href="https://twitter.com/jmschreiber91" target="_blank"><img src="logos/Twitter.png" style="height:24px"></a> 
          <a href="https://www.linkedin.com/in/jmschreiber91/" target="_blank"><img src="logos/LinkedIn.png" style="height:45px"></a>
          <a href="https://scholar.google.com/citations?user=9Y8vHKUAAAAJ&hl=en" target="_blank"><img src="logos/GoogleScholar.png" style="height:24px"></a> 
        </b>
        </span>
        <span>
          <br>
          &emsp;&emsp;
          jmschr < at > cs < dot > washington < dot > edu
        </span>

      </p>
      <hr>

      <br><br>
      <p style="font-size:16px">I am a post-doc and IGERT big data fellow in the Genome Science department at the University of Washington. Previously, I was a graduate student in the Computer Science and Engineering Department. My research focus is on the application of machine learning methods, primarily deep learning ones, to the massive amount of data being generated in the field of genome science. My research projects have involved predicting the three dimensional structure of the genome using convolutional neural networks and learning a latent representation of the human epigenome using deep tensor factorization. These projects typically involve hundreds of millions to billions of samples, making them markedly "big data" problems. I routinely contribute to the Python open source community as the core developer of pomegranate, a package for flexible probabilistic modeling, apricot, a package for data summarization for machine learning, and in the past as a core developer for the scikit-learn project. Future projects include one day being able to go outside again.  </p>

      <p style="font-size:12px">
        Machine Learning | Submodular Optimization | Big Data | Open Source Software | Computational Biology | Functional Genomics | Epigenomics
      </p>

      <p>
        <a style="text-decoration:none;color:white" href="https://github.com/jmschrei/apricot/"          class="projbutton button6" target="_blank"> apricot </a>
        <a style="text-decoration:none;color:white" href="https://github.com/jmschrei/pomegranate"       class="projbutton button1" target="_blank"> pomegranate </a>
        <a style="text-decoration:none;color:white" href="https://github.com/jmschrei/avocado/"          class="projbutton button2" target="_blank"> Avocado </a>
        <a style="text-decoration:none;color:white" href="https://github.com/jmschrei/rambutan"          class="projbutton button3" target="_blank"> Rambutan </a>
        <a style="text-decoration:none;color:white" href="https://github.com/scikit-learn/scikit-learn"  class="projbutton button4" target="_blank"> scikit-learn </a>
        <a style="text-decoration:none;color:white" href="https://github.com/jmschrei/PyPore"            class="projbutton button5" target="_blank"> PyPore </a>
      </p>

      <hr>
      <p>
        <iframe width="250" height="170" src="https://www.youtube.com/embed/WzhHVYsUGtM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
        <iframe width="250" height="170" src="https://www.youtube.com/embed/kZPXbZT5stI" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe> 
        <iframe width="250" height="170" src="https://www.youtube.com/embed/kw7Jb_Pfn4g" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
        <iframe width="250" height="170" src="https://www.youtube.com/embed/SCFsftX1RJs" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
      </p>

    </div>

    <a name="publications"><div class="jumbotron"><a></a>
      <div class="page-header">
        <h2 style="font-size:26px"><b>Selected Publications</b></h2>

        <p style="font-size:14px">
          apricot: Submodular selection for data summarization in Python, <i>arXiv</i>, 2019 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, J. Bilmes, and W.S. Noble</span><br>
          <span style="font-size:12px">
            [<a href="https://arxiv.org/abs/1906.03543" target="_blank">paper</a>]
          </span>
        </p>

        <p style="font-size:14px">
          A Genome-wide Framework for Mapping Gene Regulation via Cellular Genetic Screens, <i>Cell</i>, 2019 <br>
          <span style="font-size:12px">M. Gasperini, A.J. Hill, J.L. McFaline-Figueroa, B. Martin, S. Kim, M.D. Zhang, D. Jackson, A. Leith, <b>J. Schreiber</b>, W.S. Noble, C. Trapnell, N. Ahituv, and J. Shendure</span><br>
          <span style="font-size:12px">
            [<a href="https://www.sciencedirect.com/science/article/pii/S009286741831554X" target="_blank">paper</a>]
          </span>
        </p>


        <p style="font-size:14px">
          Completing the ENCODE3 compendium yields accurate imputations across a variety of assays and human biosamples <i>bioRxiv</i>, 2019 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, J. Bilmes, and W.S. Noble</span> <br>
          <span style="font-size:12px">
            [<a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/29/533273.full.pdf" target="_blank">paper</a>]
            [<a href="https://www.github.com/jmschrei/avocado" target="_blank">code</a>]
          </span>
        </p>

        <p style="font-size:14px">
          A pitfall for machine learning methods aiming to predict across cell types, <i>bioRxiv</i>, 2019 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, R. Singh, J. Bilmes, and W.S. Noble</span> <br>
          <span style="font-size:12px">
            [<a href="https://www.biorxiv.org/content/biorxiv/early/2019/01/04/512434" target="_blank">paper</a>]
          </span>

        </p>
        
        <p style="font-size:14px">
          Massively parallel profiling and predictive modeling of the outcomes of CRISPR/Cas9-mediated double-strand break repair, <i>Nucleic Acids Research</i>, 2019 </a><br>
          <span style="font-size:12px">W. Chen, A. McKenna, <b>J. Schreiber</b>, Y. Yin, V. Agarwal, W.S. Noble, and J. Shendure</span> <br>
          <span style="font-size:12px">
            [<a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz487/5511473 target="_blank">paper</a>]
          </span>
        </p>
        
        <p style="font-size:14px">
          Multi-scale Deep Tensor Factorization Learns a Latent Representation of the Human Epigenome, <i>bioRxiv</i>, 2019 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, T. Durham, J. Bilmes, and W.S. Noble</span> <br>
          <span style="font-size:12px">
            [<a href="https://www.biorxiv.org/content/early/2018/07/08/364976" target="_blank">paper</a>]
            [<a href="https://www.github.com/jmschrei/avocado" target="_blank">code</a>]
            [<a href="https://www.youtube.com/watch?v=kw7Jb_Pfn4g" target="_blank">talk</a>]
            [<a href="https://noble.gs.washington.edu/proj/avocado/Avocado_ISCB_2018_NoAnimation.pdf" target="_blank">slides</a>]
            [<a href="https://github.com/jmschrei/avocado/blob/master/Avocado%20Training%20Demo.ipynb" target="_blank">tutorial</a>]
          </span>
        </p>
        
        <p style="font-size:14px">
          pomegranate: Fast and Flexible Probabilistic Modeling in Python, <i>Journal of Machine Learning Research</i> 2017 <br>
          <span style="font-size:12px"><b>J. Schreiber</b></span> <br>
          <span style="font-size:12px">
            [<a href="http://www.jmlr.org/papers/volume18/17-636/17-636.pdf" target="_blank">paper</a>]
            [<a href="https://www.github.com/jmschrei/pomegranate" target="_blank">code</a>]
            [<a href="https://www.youtube.com/watch?v=kZPXbZT5stI" target="_blank">talk</a>]
            [<a href="https://github.com/jmschrei/pomegranate/blob/master/slides/pomegranate%20ODSC%20West%202018.pdf" target="_blank">slides</a>]
            [<a href="https://github.com/jmschrei/pomegranate/tree/master/tutorials" target="_blank">tutorial</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Finding the optimal Bayesian network given a constraint graph, <i>PeerJ Computer Science</i> 2017 <br>
          <span style="font-size:12px"><b>J. Schreiber</b> and W.S. Noble</span> <br>
          <span style="font-size:12px">
            [<a href="https://peerj.com/articles/cs-122/" target="_blank">paper</a>]
            [<a href="https://www.github.com/jmschrei/pomegranate" target="_blank">code</a>]
            [<a href="https://github.com/jmschrei/pomegranate/blob/master/tutorials/B_Model_Tutorial_4b_Bayesian_Network_Structure_Learning.ipynb" target="_blank">tutorial</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture, <i>bioRxiv</i>, 2017 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, M. Libbrecht, J. Bilmes, and W.S. Noble</span> <br>
          <span style="font-size:12px">
            [<a href="https://www.biorxiv.org/content/10.1101/103614v5" target="_blank">paper</a>]
            [<a href="https://www.github.com/jmschrei/rambutan" target="_blank">code</a>]
            [<a href="https://www.youtube.com/watch?v=SCFsftX1RJs" target="_blank">talk</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Segmentation of Noisy Signals Generated by a Nanopore, <i>bioRxiv</i>, 2015 <br>
          <span style="font-size:12px"><b>J. Schreiber</b> and K. Karplus</span> <br>
          <span style="font-size:12px">
            [<a href="http://biorxiv.org/content/early/2015/01/23/014258" target="_blank">paper</a>]
            [<a href="https://github.com/jmschrei/PyPore" target="_blank">code</a>]
            [<a href="https://github.com/jmschrei/PyPore/blob/master/examples/PyPore%20Tutorial.ipynb" target="_blank">tutorial</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Analysis of Nanopore Data Using Hidden Markov Models, <i>Bioinformatics</i>, 2015 <br>
          <span style="font-size:12px"><b>J. Schreiber</b> and K. Karplus</span> <br>
          <span style="font-size:12px">
            [<a href="http://bioinformatics.oxfordjournals.org/content/early/2015/02/02/bioinformatics.btv046" target="_blank">paper</a>]
            [<a href="https://github.com/jmschrei/PyPore" target="_blank">code</a>]
            [<a href="https://github.com/jmschrei/PyPore/blob/master/examples/PyPore%20Tutorial.ipynb" target="_blank">tutorial</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Discrimination among Protein Variants Using an Unfoldase-Coupled Nanopore, <i>ACS Nano</i>, 2014 <br>
          <span style="font-size:12px">J. Nivala, L. Mulroney, G. Li, <b>J. Schreiber</b>, and M. Akeson</span> <br>
          <span style="font-size:12px">
            [<a href="http://pubs.acs.org/doi/abs/10.1021/nn5049987" target="_blank">paper</a>]
          </span>
        </p>

        <p style="font-size:14px">
          Nanopores Discriminate Among Five C5 Cytosine Variants in DNA, <i>Journal of the American Chemical Sociey</i>, 2014 <br>
          <span style="font-size:12px">Z. L. Wescoe, <b>J. Schreiber</b>, M. Akeson </span> <br>
          <span style="font-size:12px">
            [<a href="http://pubs.acs.org/doi/abs/10.1021/ja508527b" target="_blank">paper</a>]
          </span>
        </p> 

        <p style="font-size:14px">
          Error rates for nanopore discrimination among cytosine, methylcytosine, and hydroxymethylcytosine along individual DNA strands, <i>Proceedings of the National Academy of Science</i>, 2013 <br>
          <span style="font-size:12px"><b>J. Schreiber</b>, Z. L. Wescoe, R. Abu-shumays, J. T. Vivian, B. Baatar, K. Karplus, and M. Akeson</span> <br>
          <span style="font-size:12px">
            [<a href="http://www.pnas.org/content/110/47/18910.short" target="_blank">paper</a>]
          </span>
        </p>
      </div>
    </div></a>

    <a name="talks"><div class="jumbotron"><a></a>
      <div class="page-header">
        <h2 style="font-size:26px"><b>Talks</b></h2>
          <ul style="font-size:14px">
            <b>Avocado: Multi-scale deep tensor factorization learns a latent representation of the human epigenome</b> <br>
              &emsp;- Intelligent Systems for Molecular Bilogy (ISMB 2018) <br>
              &emsp;- Stanford Center for Genomics and Personalized Medicine (2018) <br>

            <br>
            <b>pomegranate: Probabilistic Modeling in Python</b> <br> 
              &emsp;- <i>ODSC East (2017, Invited)</i> <br>
              &emsp;- <i>Tesla Autopilot Maps (2017, Invited)</i> <br>
              &emsp;- Data Intelligence (2017) <br>
              &emsp;- scipy (2017) <br>
              &emsp;- UW eScience Community Seminar (2015, 2016, 2017) <br>
              &emsp;- Moore-Sloan Data Science Summit (2016, 2017) <br>
              &emsp;- Seattle Data-Analytics and Machine Learning Meetup (2016) <br> 
              &emsp;- PyData Chicago (2016) <br>

            <br>
            <b>Rambutan: Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture</b> 
              &emsp;- scipy (2017) <br>
              &emsp;- <i>UCSF (2017, Invited)</i> <br>
              &emsp;- Stanford Center for Genomics and Personalized Medicine (SCGPM 2017) <br>
              &emsp;- Great Lakes Bioinformatics Conference (2017) <br>

            <br>
            <b>Large Scale HMMs for Nanopore Data Analysis</b><br>
             &emsp;- UW eScience Community Seminar (2016) <br>
             &emsp;- UW ACMS Research Seminar (2016) <br>

            <br>
            <b><a href="lectures/Parallel_Processing_in_Python.html" target="_blank">Parallel Processing in Python</a></b>: &emsp; UW eScience (2016)
            
            <br>
            <b><a href="lectures/Accelerating_Scientific_Code.html" target="_blank">Accelerating Scientific Code</a></b>: &emsp;INRIA Parietal (2015)
            
            <br>
            <b><a href="lectures/bioinformatics.html" target="_blank">What is a bioinformatician?</a></b>: &emsp; UCSC (2013-2016, Invited)
          </ul>
      </div>
    </div></a>

    <a name="employment"><div class="jumbotron"><a></a>
      <div class="page-header">
        <h2 style="font-size:26px"><b>Employment</b></h2>

        <h2 style="font-size:20px"><b>Research Intern</b></h2>
        <h2 style="font-size:14px">
          Tesla Motors Inc. <br>
          June 2017-September 2017 (3 months)
        </h2>
        <p style="font-size:16px">
          This internship focused on exploring new ways that machine learning can improve Tesla AutoPilot, and involved processing terabytes of fleet data, doing exploratory data analysis, and building working machine learning prototypes.
        </p>

        <h2 style="font-size:20px"><b>Research Intern</b></h2>
        <h2 style="font-size:14px">
          Aspen Technology <br>
          June 2016-September 2016 (3 months)
        </h2>
        <p style="font-size:16px">
          As a member of the AspenTech data science team, my focus was on implementing scalable and efficient probabilistic models for the analysis of large amounts of data.
        </p>

        <h2 style="font-size:20px"><b>Software Engineering Intern</b></h2>
        <h2 style="font-size:14px">
          Parietal, INRIA Saclay <br>
          June 2015-September 2015 (3 months)
        </h2>
        <p style="font-size:16px">
          As a software engineering intern on the scikit-learn team, my focus was on speeding up the gradient boosting implementation. My work ended up speeding up most tree-based estimators in addition to gradient boosting.
          trees.
        </p>

        <h2 style="font-size:20px"><b>Junior Specialist</b></h2>
        <h2 style="font-size:14px">
          Nanopore Group, University of California, Santa Cruz <br>
          July 2013-September 2014 (1 year 2 months)
        </h2>
        <p style="font-size:16px">
          Worked as an on-hand computational specialist to the UCSC Nanopore Group. 
          This involved me using a variety of machine learning methods (Random Forests, 
          SVMs, HMMs predominately) to analyze large volumes of data and to automate 
          previously hand-done analyses, increasing speed and accuracy. I maintained a 
          MySQL database of experimental metadata, results, summaries, and other information 
          in a normal-form compliant manner. Lastly, I was involved with writing several 
          papers based on my findings, a grant which got accepted from the NIH, and in general 
          explaining my computational work to others without my background.
        </p>

        <h2 style="font-size:20px"><b>Undergraduate Research Assistant</b></h2>
        <h2 style="font-size:14px">
          Nanopore Group, University of California, Santa Cruz <br>
          January 2011-June 2013 (2 years 6 months)
        </h2>
        <p style="font-size:16px">
          I performed both wetlab and computational research with the UCSC Nanopore Group. 
          This involved me purifying DNA, running nanopore experiments, and analyzing them 
          using simple machine learning methods. This resulted in a first author paper in 
          which I used supervised learning methods to show that the MspA nanopore can distinguish 
          between epigenetic modifications of the cytosine nucleotide, and described error rates 
          for this classification.
        </p>
      </div>
    </div></a>

</body>
</html>